ptolemy-py/python/ptolemy_client/v1/client.py

```diff
@@ -94,6 +55,20 @@ def trace(
             environment=environment,
         )
 
+def connect(base_url: str) -> Ptolemy:
+    """
+    Connect to Ptolemy client.
+    """
+    client = RecordExporter(base_url)
+    workspace_id, workspace_name = client.get_workspace_info()
+
+    return Ptolemy(
+        base_url=base_url,
+        client=RecordExporter(base_url),
```

Comment:
shouldn't you just use the client instance you created before?

Reply:
<img width="531" height="200" alt="Image" src="https://github.com/user-attachments/assets/2e7fa0ad-edf6-4d8b-88e7-2ec4ab399105" />